src/googleError.ts

```diff
@@ -78,10 +78,19 @@ export class GoogleErrorDecoder {
 
     // google.rpc.Status contains an array of google.protobuf.Any
     // which need a special treatment
+    const details: Array<protobuf.Message> = [];
+    for (const detail of status.details) {
+      try {
+        const decodedDetail = this.decodeProtobufAny(detail);
+        details.push(decodedDetail);
+      } catch (err) {
+        // cannot decode detail, likely because of the unknown type - just skip it
```

Comment:
Is there any other way we could decode this information?  Even if it's not pretty, giving the user some fighting chance to see what's returned, even a stringified object maybe, could be pretty nice.

Reply:
Users should still get a binary buffer, as before. We'll take a look how we can improve it, but for now let's just put out the fire :)